doc/appendix-de-kassensichv/terminology/terminology.md

```diff
@@ -2,7 +2,16 @@
 
 | term | description |
 |---|---|
-| Action (DE: Vorgang) | an "action" within the meaning of the KassenSichV is a related recording process of an electronic recording system. An action can comprise one or more business-actions as well as other procedures, occurrences and events. Each action must trigger at least one transaction logged by the certified technical safety device (DE: TSE).  |
+| Action (DE: Vorgang) | an "action" within the meaning of the KassenSichV is a related recording process of an electronic recording system. An action can comprise one or more business-actions as well as other procedures, occurrences and events (DE: andere Vorgänge). Each action must trigger at least one transaction logged by the certified technical safety device (DE: TSE).  |
```

Comment:
Add e.g. "Therefore, in case of long-lasting processes (e.g. in gastronomy) an order as well as a repeat order and not only the final payment process has to be sent to the ft.middleware."